ftplugin/php/phpfolding.vim

```diff
@@ -181,10 +181,10 @@ function! s:PHPCustomFolds() " {{{
 	"       'g:searchEmptyLinesPostfixing'..
 
 	" Fold function with PhpDoc (function foo() {})
-	call s:PHPFoldPureBlock('\(^\|\s\)function\s\+[A-Za-z_][A-Za-z0-9_]*\s*(', s:FOLD_WITH_PHPDOC)
+	call s:PHPFoldPureBlock('^\(public\|private\|protected\|static\|\s\)*function\s\+[A-Za-z_][A-Za-z0-9_]*\s*(', s:FOLD_WITH_PHPDOC)
```

Comment:
@rayburgemeestre This previously matched every `function` that has a space in front which is not good. It also included `abstract function` which broke the call below. Now it matches only lines with a proper function definition, with any of the visibility and static declarations in any order.

Reply:
Lines matched:

```
function a()
   function b()
public function c()
   public function d()
public        static    function e()
  static    protected function f()
// ...
```